src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/DependencyResolution/RefundsRegistry.cs

```diff
@@ -16,9 +14,10 @@ public RefundsRegistry(Type taskType, ContextWrapper contextWrapper)
             Scan(
                 scan =>
                 {
-                    scan.AssemblyContainingType<RefundsRegistry>();
+                    scan.AssembliesFromApplicationBaseDirectory();
```

Comment:
This may be more dangerous than explicitly stating  which assemblies to scan. This will include other assemblies which don't belong to the application. Would it not be safer and quicker to scan only the assemblies which know we need to do IoC on?